internal/cli/opsmanager/clusters/shutdown.go

```diff
@@ -71,24 +76,42 @@ func (opts *ShutdownOpts) Confirm() error {
 	if opts.confirm {
 		return nil
 	}
+
+	shutdownProcess := opts.clusterName
+
+	if len(opts.processes) > 0 {
+		shutdownProcess = fmt.Sprintf("%s (%s)", opts.clusterName, strings.Join(opts.processes, ", "))
+	}
 	prompt := &survey.Confirm{
-		Message: fmt.Sprintf("Are you sure you want to shutdown: %s", opts.name),
+		Message: fmt.Sprintf("Are you sure you want to shutdown: %s", shutdownProcess),
 	}
 	return survey.AskOne(prompt, &opts.confirm)
 }
 
-// mongocli cloud-manager cluster(s) shutdown <name> --projectId projectId [--force].
+func (opts *ShutdownOpts) validateInputs() error {
```

Comment:
do you need this? cluster name is always present and validated and processes are optional

Reply:
ops, left over